### PR TITLE
Change docstring style from Google to Sphinx

### LINKED ISSUE #90: Change docstring style from Google to Sphinx
**Is your feature request related to a problem? Please describe.**
The documentation for this project uses Google style docstrings. The ArchivesSnake library uses Sphinx style docstrings to generate documentation. In order to contribute the code from this project to the ArchivesSnake library, the docstrings should be updated.

**Describe the solution you'd like**
Docstrings should be changed to match Sphinx syntax.

### PR DESCRIPTION
**Pull request by a-berish**
## Changes
- fixes #90 
- remove napoleon extension from docs config
- update readme to remove reference to napoleon
- update docs strings to use Sphinx
- change args from JSONModelObject to dict (where applicable) 

## Questions
None

## Blockers
None

### CODE REVIEW

File: rac_aspace/data_helpers.py [hunk 1]
```diff
@@ -160,12 +154,11 @@ def closest_value(archival_object, key):
     Starts with an archival object, and iterates up through it"s ancestors
```
Comment (helrond) [inline rac_aspace/data_helpers.py]:
```suggestion
    Starts with an archival object, and iterates up through its ancestors
```

**Review by helrond — CHANGES_REQUESTED** [review]
Wow this looks nice! I just added one typo I noticed.

Otherwise, two things:
1. We're missing a docstring for `rac_aspace.serializers.BaseSerializer.read_data()`. That's probably my bad, but would you all be able to add that?
2. in the documentation for the CSVSerializer and TSVSerializer I'm seeing the properties showing up.
Screen Shot 2020-04-28 at 2 33 39 PM
Is there anything we can do about that?

**Review by helrond — APPROVED** [review]

**Comment by helrond** [comment]
@a-berish just need to make sure linting passes.

**Review by p-galligan — APPROVED** [review]
Approved based on linting fix.

**Pull request merged**